elevate-cpanel

```diff
@@ -1456,7 +1456,7 @@ sub backup_ea4_profile () {
 
     update_stage_file( { ea4 => $data } );
 
-    return 1;
+    return $json;
```

Comment:
note: the variable name `$json` is confusing, this is not part of these  changes
but could good to rename it as `ea_json_profile_file` or similar so we know this is a file

Reply:
Changed to `$json_path`, if that's any better.